### PR TITLE
Upgraded kafka version with new tag

### PR DESCRIPTION
**Pull request by tygrash**
Activated 0.10.2 kafka version profile and incremented the tag version.

### CODE REVIEW

**Comment by tygrash** [comment]
@HenryCaiHaiying please review and suggest modifications required, if any! Thank You

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.pinterest</groupId>
     <artifactId>secor</artifactId>
-    <version>0.23-SNAPSHOT</version>
+    <version>0.24-SNAPSHOT</version>
```
Comment (HenryCaiHaiying) [inline pom.xml]: Don't change this version line, I will handle the version upgrade.

Comment (tygrash) [inline pom.xml]: Reverted version line change.

**Comment by HenryCaiHaiying** [comment]
I will cut out a version / tag once the build passed.

**Pull request merged**